site/content/blog/2017-08-07-the-easiest-way-to-get-started.md

```diff
@@ -52,7 +52,7 @@ npm run dev
 
 This will create a new project in the `my-svelte-project` directory, install its dependencies, and start a server on http://localhost:5000.
 
-You can find more information about using TypeScript [here](blog/svelte-and-typescript).
+You can find more information about using TypeScript [here](svelte-and-typescript).
```

Comment:
```suggestion
You can find more information about using TypeScript [here](/blog/svelte-and-typescript).
```

I think we'd want this to be absolute, we have been fixing these links as absolute links recently. cc @ignatiusmb